Modelica/package.mo

```diff
@@ -2295,6 +2295,7 @@ Graphical illustrations shall not be added in the diagram layer, but can be adde
 </ul></li>
 <li> These packages should appear in the listed order.</li>
 </ol>
+<p>Other Modelica libraries may also take advantage of these guidelines, for example by referencing this documentation as <code>modelica:/Modelica.UsersGuide.Conventions#info</code>, or by using it as a source of inspiration for an independent set of guidelines.</p>
```

Comment:
Why not a proper href link to the Modelica URI? Or why not linking the resolved (but version pinned one) at https://doc.modelica.org/Modelica%204.0.0/Resources/helpOM/Modelica.UsersGuide.Conventions.html.

Reply:
I preferred to just make it a linked Modelica URI in order to stay completely tool agnostic.  Good now?

Reply:
I also think the internal reference is preferable; not to be tool agnostic but to avoid dependency on possibly changing web-sites.

Reply:
@beutlich, OK to resolve?